install_prerequisites.py

```diff
@@ -180,6 +184,11 @@ def main():
                 'curl-minimal',
             ])
 
+            # Starting with EL9, redhat-lsb-core is in the devel repo, which we don't want to
+            # leave enabled.
+            cmd = ['sudo', 'dnf', '--enablerepo=devel', 'install', '-y', 'redhat-lsb-core']
+            build.run_cmd(cmd, check_rc='dnf install redhat-lsb-core failed')
```

Comment:
So the devel repo is only enabled for the duration of this one command?

Reply:
That's correct.